game/plugin/skills/herblore/src/herblore.plugin.kts

```diff
@@ -0,0 +1,33 @@
+import org.apollo.game.message.impl.ItemOptionMessage
+import org.apollo.game.message.impl.ItemOnItemMessage
+
+on { ItemOptionMessage::class }
+        .where { option == 1 }
+        .then {
+            val unidentified = Herb[id] ?: return@then
+            it.startAction(IdentifyHerbAction(it, slot, unidentified))
+            terminate()
+        }
+
+on { ItemOnItemMessage::class }
+        .where { (id == PESTLE_MORTAR || id == KNIFE) && (targetId == PESTLE_MORTAR || targetId == KNIFE) }
```

Comment:
This check is wrong - will only pass if _both_ `id` and `targetId` are a knife or a pestle & mortar.